projects/igniteui-angular/src/lib/core/styles/components/query-builder/_query-builder-theme.scss

```diff
@@ -115,6 +118,18 @@
         .ig-typography__h6 {
             margin-bottom: 0;
         }
+
+        @if $variant != 'indigo-design' {
+            /* stylelint-disable scss/at-extend-no-missing-placeholder */
+            @extend .ig-typography__h6;
+        } @else {
+            @extend .ig-typography__h5;
+            /* stylelint-enable scss/at-extend-no-missing-placeholder */
+
+            border-top-left-radius: border-radius(rem(10px));
+            border-top-right-radius: border-radius(rem(10px));
+            border-block-end: rem(1px) solid var-get($theme, 'header-border');
```

Comment:
That header-border should be available in all themes not just in Indigo, we should not have properties available only to a specific theme.